client/asset/btc/btc.go

```diff
@@ -1021,7 +1072,7 @@ func (a amount) String() string {
 // via feeRateWithFallback.
 func (btc *baseWallet) targetFeeRateWithFallback(confTarget, feeSuggestion uint64) uint64 {
 	feeRate, err := btc.estimateFee(btc.node, confTarget)
-	if err == nil {
+	if err == nil && feeRate > 0 {
 		btc.log.Tracef("Obtained local estimate for %d-conf fee rate, %d", confTarget, feeRate)
 		return feeRate
```

Comment:
What do you think about an inner condition for `feeRate > feeRateLimit`?  To guard against the horrible anomalous behavior of `estimatefee` where it suggests more than 5 DOGE/kB.  https://github.com/decred/dcrdex/pull/1558#discussion_r850061882